podcastfy/aiengines/llm/base.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+from typing import List, Tuple
+
+from podcastfy.core.character import Character
+
+
+class LLMBackend(ABC):
```

Comment:
Here we should define a common interface for our LLM (content generator) models.
It is best practice today to have OpenAI as the standard interface.

Reply:
Sounds like a good idea! That would help with tons of things (like multimodality) can you develop or give me a pointer to the interface you had in mind?